documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
+
 ## Prerequisites
 
 Ensure the following tools are installed:
 
 - `bito` : https://github.com/gitbito/CLI
 
-- `mermaidcli` : https://github.com/mermaid-js/mermaid-cli
+- `Code2Flow:` : https://github.com/scottrogowski/code2flow
```

Comment:
**Suggestion**: Update the documentation to include a section on installing and configuring Code2Flow and Graphviz, as these are now dependencies for the tool. Providing installation instructions and configuration tips would help users set up their environment for successful documentation generation. <br> **Code Suggestion**: 
 ```
 - 'Code2Flow:' : https://github.com/scottrogowski/code2flow. To install Code2Flow, follow the instructions on the GitHub page. Ensure Code2Flow is accessible in your PATH.
 - 'Graphviz (dot):' : https://graphviz.org/download/. Graphviz is required for converting Code2Flow diagrams into visual formats. Follow the installation instructions for your operating system and ensure the 'dot' command is accessible in your PATH.
 ```